gcp_pilot/drive.py

```diff
@@ -0,0 +1,111 @@
+# Reference: https://developers.google.com/drive/api/reference/rest/v3
+
+from typing import Any
+
+from gcp_pilot.base import DiscoveryMixin, GoogleCloudPilotAPI
+
+
+class Drive(GoogleCloudPilotAPI):
+    _scopes = ["https://www.googleapis.com/auth/drive"]
+
+    def __init__(self, **kwargs):
+        super().__init__(serviceName="drive", version="v3", **kwargs)
+
+
+class Permissions(DiscoveryMixin, Drive):
+    @property
+    def get_service(self):
```

Comment:
Permissions is actually a [resource](https://developers.google.com/drive/api/reference/rest/v3/permissions).

And a property with a verb looks weird, so I'd suggest:

```suggestion
    @property
    def _resource(self):
```